### DISCUSSION TITLE
Can we add -fPIC in CFLAGS thus the static library can be linked into other shared libraries?

### DISCUSSION THREAD

**Discussion by levyfan** [protobuf-c/protobuf-c#608] [Q&A]
Linking with libprotobuf-c.a will cause `relocation R_X86_64_32 ...` error currently. Can we add `-fPIC` in the configure.ac file?

**Comment by lipnitsk** [comment]
The library is configured to be built as a standalone package. You are welcome to alter the configuration or use your own toolchain to accomplish what you need, however that is beyond the scope of this project.

**Comment by hu6360567** [comment]
Just SET envirnoment variable `CFLAGS=-fPIC` when running `configure`
